docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation.md

```diff
@@ -0,0 +1,152 @@
+---
+title: How to create example cases for a Custom Expectation
+---
+import Prerequisites from '../creating_custom_expectations/components/prerequisites.jsx'
+
+This guide will help you add example cases to document and test the behavior of your [Expectation](../../../reference/expectations/expectations.md). 
+
+<Prerequisites>
+
+ - Created a [Custom Expectation](../creating_custom_expectations/overview.md)
+
+</Prerequisites>
+
+
+Example cases in Great Expectations serve a dual purpose:
+* They help the users of the Expectation understand its logic by providing examples of input data that the Expectation will evaluate;
+* And provide test cases that the Great Expectations testing framework can execute automatically.
+
+If you decide to contribute your Expectation, its entry in the [Expectations Gallery](https://greatexpectations.io/expectations/) will render these examples.
+
+We will explain the structure of these tests using the Custom Expectation implemented in [How to create Custom Column Aggregate Expectations](../creating_custom_expectations/how_to_create_custom_column_aggregate_expectations.md).
+
+### Steps
+
+#### 1. Decide which tests you want to implement
+
+Expectations can have a robust variety of possible applications. We want to create tests that demonstrate -- and verify -- the capabilities and limitations of our Custom Expectation.
+
+<details>
+  <summary>What kind of tests can I create?</summary>
+These tests can include examples intended to pass, fail, or error out, and expected results can be as open-ended as <inlineCode>{`{"success": False}`}</inlineCode>, or as granular as:
+<code>{`{
+  "success": True,
+  "expectation_config": {
+      "expectation_type": "expect_column_value_z_scores_to_be_less_than",
+      "kwargs": {
+          "column": "a",
+          "mostly": 0.9,
+          "threshold": 4,
+          "double_sided": True,
+      },
+      "meta": {},
+  },
+  "result": {
+      "element_count": 6,
+      "unexpected_count": 0,
+      "unexpected_percent": 0.0,
+      "partial_unexpected_list": [],
+      "missing_count": 0,
+      "missing_percent": 0.0,
+      "unexpected_percent_total": 0.0,
+      "unexpected_percent_nonmissing": 0.0,
+  },
+  "exception_info": {
+      "raised_exception": False,
+      "exception_traceback": None,
+      "exception_message": None,
+  }
+}`}
+</code>
+</details>
+
+At a minimum, we want to create tests that show what our Custom Expectation will and will *not* do. 
+These basic positive and negative example cases are the minimum amount of test coverage required for a Custom Expectation to be accepted into the Great Expectations codebase at an [Experimental level](../../../contributing/contributing_maturity.md#contributing-expectations).
+To begin with, let's implement those two basic tests; one positive example case, and one negative example case. 
```

Comment:
```suggestion
To begin with, let's implement those two basic tests: one positive example case, and one negative example case. 
```
A ; is usually only appropriate if it can be replaced with the word "and" and have the sentence still make sense.  On the other hand, a colon can be used to denote that the following is a list of some sort.